content/post/articles/Become-A-Software-Developer.md

```diff
@@ -0,0 +1,106 @@
+
+###### Introduces new career interests :
+  Some college degree programs require students to take a range of general courses across different subject areas. During this process, students may discover a potential alternative career path that interests them. For example, a student who wants to major in education may decide to minor in math to work as a math teacher.
+  Helps market candidates to employers: Having a college degree can help candidates standout to employers when they include their degree details on their resume. A college degree shows employers that candidates have the ability to learn and excel with multiple priorities (courses) at a time.
+
+
+##### #Cons :
+
+###### Enhances student loan debts : 
+  Some students may have the financial means to minimize student loan debts that they need to pay off after college. However, many students look to financial aide options to afford their education and student housing. Therefore,  earning a college degree could cause students to incur student loan debt that they have to pay off over time.
+
+###### Limits ability to switch careers : 
+  Because college degrees showcase a student's ability to excel in a particular industry or career field, it might affect their ability to switch careers quickly. They may also need to earn additional degrees or certifications to qualify for other career options.
+
+###### Minimizes work experience during degree programs : 
+  Although some degree programs offer internships and work experience, a lot of degree programs consist mainly of in-class instruction. This potentially limits the amount of real-world professional experience that could help prepare students for life after their degree.
+
+#  BootCamps
+
+Coding Bootcamps offer crash courses in software development. They are intensive and accelerated learning programs that equip learners with skills in areas
+such as Full-Stack Web Development, Data Science, Digital Marketing, UX / UI Design and more.
+Bootcamps usually take 12 weeks (full time), it can also be up to 24 weeks (part time).
+
+##### #Pros :
+
+###### Cheaper and Quicker Than a College Degree :
+ Save money and get it done faster? One of the major draws for attending coding bootcamp is its cost effectiveness and the amount of time it takes to graduate.
+ Besides being good for your wallet, bootcamp allows you to get all your coursework done in about 12 weeks for full-time students or 24 weeks for part-time, rather than four years.
+
+###### Practical (Hands-On) Curriculum :
+ Technology is constantly changing—and so must the curriculum. Coding bootcamps offer industry-inspired curriculum that’s constantly updated and applicable to real jobs in the tech world.
```

Comment:
> Technology is constantly changing—and so must the curriculum

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^

Why the double hyphen here?

Reply:
a mistake, i wrote too fast :).